### DISCUSSION TITLE
vcpkg stuck in manifest mode, refuses to install packages manually

### DISCUSSION THREAD

**Discussion by Be-ing** [microsoft/vcpkg#19401] [Q&A]
**Describe the bug**
I used manifest mode locally as a test for one project. Then I switched back to a branch I was working on for packaging a new port. However when I tried to use `vcpkg install`, I got an error:
```
vcpkg on  libsbsms 
❯ ./vcpkg install libsbsms
Error: 'install' requires 0 arguments, but 1 were provided.
```

I tried re-bootstrapping, deleting the whole vcpkg Git repository and recloning it, `rm -rf ~/.vcpkg ~/.cache/vcpkg`, but none of those worked. I don't know where the broken state is stored or how I can fix it.

**Environment**
- OS: Linux (Fedora 34)
- Compiler: GCC

**To Reproduce**
1. Build a CMake project in manifest mode
2. ./vcpkg install xxxx
2. See error

**Expected behavior**
`vcpkg install` would work.

**Comment by PhoebeHui** [comment]
@Be-ing, since you are in manifest mode, so could you try './vcpkg install' instead? it would install the ports configured in your vcpkg.json file.

**Reply by Be-ing** [reply]
I have been doing that now. Manifest mode is nice for building applications with vcpkg but it is cumbersome for working on packaging ports.

**Comment by cenit** [comment]
is there any vcpkg.json file anywhere? that tricks vcpkg into manifest mode..,

**Comment by Neumann-A** [comment]
So I also had that problem when I was using vcpkg as a submodule in my projects. The reason is that vcpkg looks into parent directories for a `vcpkg.json` to switch to manifest mode. As such I moved my `vcpkg.json` from `CMAKE_SOURCE_DIR` to `CMAKE_SOURCE_DIR/vcpkg_manifest` to avoid vcpkg in the submodule to pick up the `vcpkg.json` from my project

**Reply by Be-ing** [reply]
My local vcpkg repo is adjacent to my other repos:
`~/sw/vcpkg`
`~/sw/tenacity`
`~/sw/portaudio`

**Comment by Be-ing** [comment]
https://vcpkg.io/en/docs/users/manifests.html documents that setting the environment variable VCPKG_MANIFEST_MODE to OFF should disable manifest mode, but this does not work for me.

**Comment by Be-ing** [comment]
Does anyone have other ideas? It is pretty cumbersome to work on PRs to send upstream with this.

**Reply by autoantwort** [reply]
You can pass the flag `--feature-flags=-manifests`

**Reply by Be-ing** [reply]
That works, thanks!
